src/sonic-config-engine/sonic-cfggen

```diff
@@ -34,6 +34,18 @@ def is_ipv6(value):
             return False
     return addr.version == 6
 
+def get_machine_info():
+    if not os.path.isfile('/host/machine.conf'):
```

Comment:
should fail explicitly when there is no machine.conf since we do not which platform it is now.